doc/cmake.txt

```diff
@@ -209,6 +210,32 @@ cq			Close the CMake console window.
 
 <C-C>			Stop the current command.
 
+==============================================================================
+Events							*cmake-events*
+
+To customize behaviour after a command has finished, Vim-CMake calls user
+autocommands
+
+List of events~
+
+CMakeBuildFailed
```

Comment:
Please format like e.g. the Global <Plug> Mappings, i.e. event name on the left, and small description on the right (indented like in the <Plug> Mappings as well)